editor/plugins/script_editor_plugin.cpp

```diff
@@ -2613,8 +2613,8 @@ ScriptEditor::ScriptEditor(EditorNode *p_editor) {
 
 	tab_container = memnew(TabContainer);
 	tab_container->set_tabs_visible(false);
+	tab_container->set_custom_minimum_size(Size2(200 * EDSCALE, 0));
```

Comment:
there had been this custom minimum size.
but, IIRC, it was removed by intent.

Reply:
maybe it was `Tab` for script list. hm...

Reply:
anyway this makes another glitch as what your gif shows.

Reply:
The gif is from before the change